xmtp_content_types/src/reaction.rs

```diff
@@ -47,6 +48,41 @@ impl ContentCodec<ReactionV2> for ReactionCodec {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct LegacyReaction {
```

Comment:
We can't shoe-horn the legacy reaction into the new reaction type? I thought the data was about the same

Reply:
Original type had arbitrary String for reaction action and reaction schema ([js sdk used union type to restrict it](https://github.com/xmtp/xmtp-js/blob/0f0ae21a335cbb5b1a401248ce9a080314559f5f/content-types/content-type-reaction/src/Reaction.ts#L14-L37)), I updated[ to enum](https://github.com/xmtp/proto/blob/e295b4666d8f71469b953c7288cd34beea0e40d9/proto/mls/message_contents/content_types/reaction.proto#L18-L31) in the new proto type to be more precise so we wouldnt need to do this type of conversion across the SDKs: https://github.com/xmtp/xmtp-android/blob/51c7b170d53f73d32d3a1d33b0d170d156183e43/library/src/main/java/org/xmtp/android/library/codecs/ReactionCodec.kt#L40-L55

Reply:
Ahh. Too bad. 

Agree the new format is nicer.